src/StackExchange.Exceptional.AspNetCore/ConfigSettings.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using StackExchange.Exceptional.Internal;
+using StackExchange.Exceptional.Notifiers;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+
+namespace StackExchange.Exceptional
+{
+    /// <summary>
+    /// The Settings element for Exceptional's configuration.
+    /// This is the legacy web.config settings, that only serve as an adapter to populate <see cref="Settings"/>.
+    /// </summary>
+    public partial class ConfigSettings
+    {
+        const string CONFIGSECTION_KEY = "Exceptional";
+        private IConfigurationSection _exceptionalConfiguration;
+
+        //TODO: We really need to add a settings validation
+        public static void ConfigureSettings(IConfiguration configuration, Settings settings)
+        {
+            var configSettings = new ConfigSettings();
+            configSettings._exceptionalConfiguration = configuration.GetSection(CONFIGSECTION_KEY);
+            if (configSettings._exceptionalConfiguration.Value != null)
+            {
+                configSettings._exceptionalConfiguration.Bind(configSettings);
+                configSettings.Initialize(settings);
+            }
+        }
```

Comment:
Sounds good to me. Changed in b0e02c8 Please take a look if this fits what you are talking about